src/main/java/com/faforever/client/preferences/ui/SettingsController.java

```diff
@@ -356,6 +365,17 @@ private void initNotifyMeOnAtMention() {
     notifyAtMentionDescription.setText(i18n.get("settings.chat.notifyOnAtMentionOnly.description", "@" + username));
   }
 
+  private void initAllowReplaysWhileInGame(Preferences preferences) {
+    JavaFxUtil.bindBidirectional(allowReplayWhileInGameCheckBox.selectedProperty(), preferences.getForgedAlliance().allowReplaysWhileInGameProperty());
+    try {
+      gameService.isGamePrefsPatchedToAllowMultiInstances()
+          .thenAccept(isPatched -> allowReplayWhileInGameButton.setDisable(isPatched));
+    } catch (IOException e) {
+      log.warn("Failed evaluating if game.prefs file is patched for multiple instances.");
```

Comment:
also log the Exception (always)
```suggestion
      log.warn("Failed evaluating if game.prefs file is patched for multiple instances.", e);
```